book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
```

Comment:
- [ ] A book has many rentals hence there should be a method for adding rentals(add_rental) in the Book class. I think this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) would be helpful to you :+1: